### PR TITLE
Feature/#21 avatar and variables

### LINKED ISSUE #29: Feature/#21 - Add avatar graphics and environment varaibles
Add graphics for avatar in resources directory.
Add environment variables for default values

### PR DESCRIPTION
**Pull request by Enessetere**

### CODE REVIEW

**Review by SebastianBabinski1 — APPROVED** [review]

**Review by SebastianBabinski1 — APPROVED** [review]
Looks good :)

File: src/app/App.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export class App {
-  init() {
+  init(): string {
```
Comment (synowa) [inline src/app/App.ts:2]: oh, ts 😄

**Review by synowa — APPROVED** [review]
Ok, thanks!

**Review by DominikNowak — APPROVED** [review]
Looks ok, but consider changing name of folder with animals images. 'avatars' is little misleading and suggest there are images for player avatar.

**Comment by synowa** [comment]
> Looks ok, but consider changing name of folder with animals images. 'avatars' is little misleading and suggest there are images for player avatar.

Hmm... it really was misleading^^ I was sure that those images were the ones with avatars for the player to choose from 😄

**Comment by synowa** [comment]
Sorry, I hope we can name a folder with actual avatars for players  "player avatars" or sth like that 😸

**Pull request merged**